client/src/pages/Modal.scss

```diff
@@ -34,28 +77,49 @@
   .subtitle {
     font-weight: normal;
     font-size: 16px;
-    line-height: 20px;  
+    line-height: 20px;
     margin-bottom: 40px;
   }
 
   .link {
-    color: #004D57;
+    color: #004d57;
```

Comment:
Could we assign `#adadad`,`#004d57`, `#7d7d7d`, and `#e9e9e9` to global scss variables in `_config.scss`?